src/components/Input/CheckBox.svelte

```diff
@@ -1,9 +1,25 @@
 <script>
+    import {FontAwesomeIcon} from 'fontawesome-svelte';
+    import {faQuestionCircle} from '@fortawesome/free-solid-svg-icons';
+    import ToolTip from '../ToolTip.svelte';
+    import configurationOptionStrings from '../../strings/configuration_options.json';
+    import {prettyPrintLabel} from "../../utils/string";
+
     export let label = '';
     export let checked = false;
 </script>
 
-<label class="flex items-center space-x-3 my-2">
-    <input type="checkbox" bind:checked={checked} name="checkbox" class="form-tick appearance-none h-5 w-5 border border-gray-300 rounded-md checked:bg-blue-600 checked:border-transparent focus:outline-none">
-    <span class="text-gray-900 font-medium text-sm">{label}</span>
-</label>
\ No newline at end of file
+<div class="flex flex-row">
+    <label class="flex items-center space-x-3 my-2">
+        <input type="checkbox" bind:checked={checked} name="checkbox"
+               class="form-tick appearance-none h-5 w-5 border border-gray-300 rounded-md checked:bg-blue-600 checked:border-transparent focus:outline-none cursor-pointer">
+        <span class="text-gray-900 font-medium text-sm">{configurationOptionStrings[label]?.title ?? prettyPrintLabel(label)}</span>
+    </label>
+    <div class="p-2">
```

Comment:
Should the `#if` be on the outside of this div element?

Reply:
good spot. resolved in later commit